web/components/issues/issue-layouts/kanban/block.tsx

```diff
@@ -124,6 +124,10 @@ export const KanbanIssueBlock: React.FC<IssueBlockProps> = memo((props) => {
   const [isDraggingOverBlock, setIsDraggingOverBlock] = useState(false);
   const [isCurrentBlockDragging, setIsCurrentBlockDragging] = useState(false);
 
+  const canEditIssueProperties = canEditProperties(issue.project_id);
```

Comment:
optional chain issue, since we yet haven't checked if it is undefined below.

Reply:
Understood, I've implemented the necessary changes.